src/api.js

```diff
@@ -28,10 +28,16 @@ const post = (path, obj) =>
     body: JSON.stringify(obj),
   });
 
+const deleteMeeting = path =>
```

Comment:
This shouldn't have the `Meeting` in it's name since it's a generic delete call.

Reply:
renamed to deleteMethod since delete is reserved.